Cue/app/setup.js

```diff
@@ -29,6 +30,7 @@ function setup(): ReactClass<{}> {
       if (this.state.isLoading) {
         return null;
       }
+      CueApi.setAuthHeader(this.state.store.getState().user.userId, this.state.store.getState().user.accessToken);
```

Comment:
Wowww k im dumb, my debugger was lying to me and had a different context. I will push a commit using callback method